README.md

```diff
@@ -26,6 +26,16 @@ We manage non-CAPZ workload clusters in CAPZ MCs too. We call this concept as `m
 Supported non-CAPZ providers:
 - CAPV WCs (Vsphere)
 
+#### Tagging Resource Groups for Non-CAPZ workload clusters
+
+For CAPZ workload clusters, the resource group of the DNS zone is managed by CAPZ.
+CAPZ allows for tagging the resource group of the DNS zone with additional tags set in the `AzureCluster` resource.
+For non-CAPZ workload clusters, the resource group of the DNS zone is managed by `dns-operator-azure`.
+The `dns-operator-azure` supports tagging the resource group of the DNS zone via prefixed annotations on the non-CAPZ workload cluster's Infrastructure Cluster resource.
+
+Annotations prefixed by `azure-resource-tag/` will be used to tag the resource group of the DNS zone.
```

Comment:
I have arbitrarily picked this prefix. Please suggest anything else you find more suitable.

Reply:
That looks good 👍